Dockerfile

```diff
@@ -70,11 +71,10 @@ COPY requirements.txt .
 
 # build xml2rfc
 RUN pip3 install -r requirements.txt \
-    "weasyprint==52.5" \
-    "pycairo<1.20" \
+    "weasyprint>=53.0" \
```

Comment:
Shouldn't this be >=55 now?

Reply:
Why >= 55?  weasyprint is actually at 56 now upstream, but there's no good reason to increase the version if you don't actually need the newer version.

Put another way: please do not gratuitously increase the minimum version needed.  if 53.0 works as well, there's no reason to have the elevated version requirements.

If there is a specific reason that 55 is necessary, then of course make it required, but otherwise, you do the ecosystem a favor by indicating the flexibility explicitly.

Reply:
I don't see any specific reason for using 55 or 56 at this point.
So I'm happy to keep `weasyprint>=53.0` wider compatibility.
But services like author tools will start using the latest available version.

Reply:
yep, that sounds entirely reasonable.  the specified version ranges are constraints, but of course any specific instance will choose whatever version of the package it wants as long as it's within the constraints.